cat_server/helloworld/unikernel.ml

```diff
@@ -3,6 +3,9 @@ open V1_LWT
 
 module Main (C:CONSOLE) = struct
 
-  let start console = OS.Time.sleep 2.0 >>
+  let start console =
+    OS.Time.sleep 2.0
+
```

Comment:
The experienced OCaml programmer parses an end of function here. Functions in OCaml do not sport blank lines (unless you have a disputable style).